CHANGELOG.md

```diff
@@ -1,3 +1,350 @@
+- Update Prometheus Operator to v0.43.2 ([#1162](https://github.com/kinvolk/lokomotive/pull/1162)).
+- Update Calico to v3.17.0 ([#1251](https://github.com/kinvolk/lokomotive/pull/1251)).
+
+#### Terraform provider updates
+
+- Update Terraform providers to their latest versions([#1133](https://github.com/kinvolk/lokomotive/pull/1133)).
+
+#### Other updates
+
+- Update aws-ebs-csi-driver to v0.7.0 ([#1135](https://github.com/kinvolk/lokomotive/pull/1135)).
+
+#### New platforms
+
+- Add support for Tinkerbell as a platform ([#392](https://github.com/kinvolk/lokomotive/pull/392)).
+
+#### Bug fixes
+
+- Fix bug adding worker pools with TLS bootstrap enabled ([#1181](https://github.com/kinvolk/lokomotive/pull/1181)).
+- contour: apply node affinity and tolerations also to certgen job ([#1161](https://github.com/kinvolk/lokomotive/pull/1161)).
+- Improve hardware reservations validation rules on Equinix Metal ([#1186](https://github.com/kinvolk/lokomotive/pull/1186)).
+- Equinix Metal: ensure Calico CRDs gets created before use ([#1177](https://github.com/kinvolk/lokomotive/pull/1177)). They were failing occasionally.
+- openebs-operator: set webhook policy to Ignore ([#1172](https://github.com/kinvolk/lokomotive/pull/1172)). This works around a bug in Kubernetes, see this [comment](https://github.com/openebs/openebs/issues/3046#issuecomment-642558701) for details.
+- Don't run control plane components as DaemonSets on single control plane node clusters ([#1193](https://github.com/kinvolk/lokomotive/pull/1193)).
+- calico-host-protection: Add custom locked down PSP configuration ([#1274](https://github.com/kinvolk/lokomotive/pull/1274)).
+
+#### Features
+
+- Add Packet CCM to Packet platform ([#1155](https://github.com/kinvolk/lokomotive/pull/1155)).
+- contour: parameterize Envoy scraping interval ([#1229](https://github.com/kinvolk/lokomotive/pull/1229)).
+- Expose --conntrack-max-per-core kube-proxy flag ([#1187](https://github.com/kinvolk/lokomotive/pull/1187)).
+- Add `require_volume_annotation` for restic plugin ([#1132](https://github.com/kinvolk/lokomotive/pull/1132)). This PR is based on top of #1131 and would need to be merged first.
+- Print bootkube journal if cluster bootstrap fails ([#1166](https://github.com/kinvolk/lokomotive/pull/1166)). This makes easier to debug problems with cluster bringup.
+- aws-ebs-csi-driver: Add dynamic provisioning, resizing, snapshot options ([#1277](https://github.com/kinvolk/lokomotive/pull/1277)). Now the user has ability to control the AWS EBS driver to enable or disable provisioning, resizing, snapshotting.
+
+## Documentation
+
+- Add OpenEBS upgrade guide ([#1163](https://github.com/kinvolk/lokomotive/pull/1163)).
+- Add Rook Ceph upgrade guide ([#1165](https://github.com/kinvolk/lokomotive/pull/1165)),
+
+#### Others
+
+- Pull control plane images from Quay ([#1226](https://github.com/kinvolk/lokomotive/pull/1226)). To avoid hitting Docker Hub pulling limits.
```

Comment:
```suggestion
- Pull control plane images from Quay to avoid hitting Docker Hub pulling limits ([#1226](https://github.com/kinvolk/lokomotive/pull/1226)).
```